lib/sshkit/backends/netssh.rb

```diff
@@ -96,6 +96,12 @@ def config
         end
       end
 
+
+      def redact(arg) # Used in execute_command to hide redact() values user passes in
+        raise ArgumentError, 'redact() does not support Array or Hash' if arg.is_a?(Array) || arg.is_a?(Hash)
+        ['*REDACTED*',arg]
```

Comment:
I anticipate that some SSHKit users will complain that `redact` works for SSH backends but not for local backends. Generally speaking we try to keep the two in parity. Is there an easy way to add the `redact` functionality to the local backend as well?